tests/plugins/pastefromword/generated/PFW_image.js

```diff
@@ -0,0 +1,39 @@
+/* bender-tags: clipboard,pastefromword */
```

Comment:
Please, rename file to lowercased `pfw_image`, actually... `pfw` prefix doesn't have much use here. After all it's a directory for pfw so only its tests are here. 🙂 So how about just `image`?